web-common/src/runtime-client/local-service.ts

```diff
@@ -0,0 +1,163 @@
+import type { PartialMessage } from "@bufbuild/protobuf";
+import { createPromiseClient } from "@connectrpc/connect";
+import { createConnectTransport } from "@connectrpc/connect-web";
+import type {
+  CreateMutationOptions,
+  CreateQueryOptions,
+} from "@rilldata/svelte-query";
+import { LocalService } from "@rilldata/web-common/proto/gen/rill/local/v1/api_connect";
+import {
+  DeployRequest,
+  DeployValidationRequest,
+  GetMetadataRequest,
+  GetVersionRequest,
+  PushToGithubRequest,
+} from "@rilldata/web-common/proto/gen/rill/local/v1/api_pb";
+import type { RpcStatus } from "@rilldata/web-common/runtime-client/gen/index.schemas";
+import type { ErrorType } from "@rilldata/web-common/runtime-client/http-client";
```

Comment:
Shouldn't it use the Connect error types instead of the ones from Orval? See https://connectrpc.com/docs/web/errors